manifest.xml

```diff
@@ -90,12 +90,10 @@
       <skill>advanced</skill>
     </tutorial>
 
-    <tutorial title="Pose frame semantics: proposed behavior" ref='pose_frame_semantics_proposal' type='proposal'>
-      <markdown version="1.7">pose_frame_semantics/proposal.md</markdown>
```

Comment:
this will break any prior links to the proposal. perhaps we can add an entry for the documentation, while keeping the proposal in place?

Reply:
@eacousineau I added the proposal back as version 1.7 and set the new tutorial version to be 1.7.1. Ideally it'd be nice to have the proposal to take its own tag `proposal` but I assume that would also break links.